vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -40,12 +41,15 @@ ReactApplication::ReactApplication(IInspectable const &outer) noexcept : ReactAp
 #ifndef USE_WINUI3
   Suspending({this, &ReactApplication::OnSuspending});
 
-#if defined _DEBUG && !defined DISABLE_XAML_GENERATED_BREAK_ON_UNHANDLED_EXCEPTION
+#if !defined DISABLE_XAML_GENERATED_BREAK_ON_UNHANDLED_EXCEPTION
   UnhandledException([this](IInspectable const &, UnhandledExceptionEventArgs const &e) {
+#if defined _DEBUG
     if (IsDebuggerPresent()) {
       auto errorMessage = e.Message();
       __debugbreak();
-    }
+    } else
+#endif
+      Mso::React::CrashManager::OnUnhandledException();
```

Comment:
ReactApplication only handles the most basic type of RNW app.  Everything ReactApplication does should be accessible to more complex apps that are not using ReactAppliction.  So we should add some kind of OnUnhandledException function to one of the IDLs.

Reply:
Do IDLs support static functions? Or do we have a static class somewhere for this purpose? I agree we need to expose it, but not sure where it belongs.